gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/TemplateMapper.java

```diff
@@ -133,16 +133,6 @@ private List<DomainResource> mapTemplate(RCMRMT030101UK04EhrExtract ehrExtract,
         return List.of(parentObservation);
     }
 
-    private void addChildReferencesToQuestionnaireResponse(QuestionnaireResponse questionnaireResponse,
```

Comment:
There is some commented code on lines 128-132 which I imagine should get deleted too?